state/modelgeneration.go

```diff
@@ -172,6 +172,55 @@ func assignGenerationUnitTxnOps(id, appName, unitName string) []txn.Op {
 	}
 }
 
+// Complete marks the generation as completed.
+func (g *Generation) Complete() error {
```

Comment:
Is there a better name for this?

Reply:
`MakeCurrent`, `Activate`, `Apply`? I think the first is good verbiage for what we are doing.

Reply:
MakeCurrent works for me.  ty.